e2etests/internal/runner/runner_signal_fixture.go

```diff
@@ -102,7 +102,7 @@ func (run *RunFixture) Run() RunResult {
 		t.Fatalf("unexpected run mode: %d", run.mode)
 	}
 
-	exec := run.cli.NewCmd(args...)
+	exec := run.cli.NewCmd(false, args...)
```

Comment:
_nitpick_ I would say to use constants for booleans here to improve readability but not important.
```suggestion
	exec := run.cli.NewCmd(noInteractive, args...)
```

Reply:
Done in [cd452d3](https://github.com/terramate-io/terramate/pull/2217/commits/cd452d3eb2d2852fcd0f68459341a07e85dd6414)